server/src/reactruntime.js

```diff
@@ -37,6 +37,7 @@ exports.HeadTemplate = class HeadTemplate extends React.Component {
       localeScripts.push(<script key={`l10n-${locale}`} src={this.props.staticLink(`/static/locales/${locale}.js`)} />);
     }
 
+    console.log("include wantsauth?", {authenticatied: this.props.authenticated, hasFxa: this.props.hasFxa, authFxa: this.props.authFxa});
     const wantsAuth = (!this.props.authenticated) ||
```

Comment:
I don't think we need authFxa here, if we fix the `/confirm-login` endpoint

Reply:
Link to PR where we added wantsauth check to conditionally include wantsauth.js on all pages https://github.com/mozilla-services/screenshots/pull/4759/files#diff-79019fb192622531c0f432b6833b083dR50

Here we are relying on auth flag from confirm-login redirect to load wantsauth.js and make explict calls to getAuthData that sets accountId cookie and update accountId request parameter used to fill hasFxA model property used in all pages to determine if a user has firefox account